homework-submission/todo_react/src/TodoLists.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./TodoLists.css";
+
+
+
+export default class TodoLists extends React.Component {
+
+  render() {
+
+    let { description, deadline, done, id, handleClick, removeItem } = this.props;
+
+    // console.log(data[2].deadline)
+
+    return (
+      <div className="list">
+
+        <ul>
+          <li>
+
+            <input type="checkbox" defaultChecked={done} onChange={handleClick} />
```

Comment:
Only if this is not a static form for a single use like a signup or similar use `defaultChecked`, But if not needed you can use `checked` instead